.idea/runConfigurations/keptn_bridge__Kube_ContDeploy_.xml

```diff
@@ -0,0 +1,15 @@
+<component name="ProjectRunConfigurationManager">
```

Comment:
Please add a .git ignore for .idea

Reply:
This is a shared run configuration which was added on purpose, same as the other ones that are already in that folder: https://github.com/keptn/keptn/tree/master/.idea/runConfigurations